brave/brave-core
# Implement promoted content ads

Resolves https://github.com/brave/brave-browser/issues/13468

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.

## Test Plan:

- Confirm users are rewarded BAT for viewing promoted content within Brave Today. The catalog contains a new ad type `promoted_cotent_all_v1` where the `creative_instance_id` has to match the Brave Today feed
- Confirm viewed confirmations are redeemed with the server (see diagnostic log and confirm with server team)
- Confirm clicked confirmations are redeemed with the server (see diagnostic log and confirm with server team)
